includes/AppLeave.php

```diff
@@ -161,6 +161,7 @@
 	//echo "Sending application to $appr";
 	
 	$destDB = str_replace('.','$',$appr);
+	$destDB = str_replace(' ','_',$destDB);
```

Comment:
__[code_sniffer]__
* Spaces must be used for alignment; tabs are not allowed
* No space found after comma in function call

<!-- Comment by [SideCI](https://sideci.com) -->